test/units/testsuite-35.sh

```diff
@@ -420,48 +420,6 @@ EOF
     fi
```

Comment:
Could you also drop crond from TEST-35/test.sh ?

Reply:
Fixed.